serverless/email/README.md

```diff
@@ -0,0 +1,88 @@
+# email
+
+This is a serverless email webhook, which handles unsubscribes, bounces, idempotency and emailing.
+
+```bash
+.
+├── Makefile                    <-- Make to automate build
+├── README.md                   <-- This instructions file
+├── webhook                     <-- Source code for the webhook handler
+│   ├── main.go                 <-- Lambda function code
+├── unsubscribe                 <-- Source code for the unsubscribe handler
+│   ├── main.go                 <-- Lambda function code
+├── status                      <-- Source code for the sns status reporting handler
+│   ├── main.go                 <-- Lambda function code
+└── template.yaml
+```
+
```

Comment:
This seems like its missing something.  Does this allow for local testing end to end?